.github/workflows/Publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 18.x
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
Where and who stores this secret such that Github can access NPM? And whose account is used for that?

Reply:
Can we somewhere document that? Probably the best option is as a comment in the script above the line.

Reply:
Too late it is already merged. :grinning: